core-java-modules/core-java-arrays-convert/src/test/java/com/baeldung/array/conversions/PrimitiveByteArrayToByteArrayUnitTest.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.array.conversions;
+
+import java.util.Arrays;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PrimitiveByteArrayToByteArrayUnitTest {
```

Comment:
let's explicitly put public here and in the test methods

Reply:
Fixed